tests/test_lookups.py

```diff
@@ -87,6 +87,17 @@ def test_call_dict(self):
 
         self.assertEqual(actual, 'bar')
 
+    def test_call_object(self):
+        """Ensure that we can lookup values in non iterable objects.
+        """
```

Comment:
thanks for the docstring!

can you please move this ending triple quote at the end of last line as per pep8:

> For one liner docstrings, please keep the closing """ on the same line.

[source](https://www.python.org/dev/peps/pep-0008/)